app/uk/gov/hmrc/tai/forms/EditIncomeForm.scala

```diff
@@ -118,7 +118,7 @@ object EditIncomeForm {
         "description" -> text,
         "employmentId" -> number,
         "newAmount" -> TaiValidator.validateTaxAmounts(Messages("error.tai.updateDataEmployment.blankValue"),
-           Messages("error.tai.updateDataEmployment.enterRealNumber"),
+           Messages("error.invalid.monetaryAmount.format.invalid"),
```

Comment:
These should be using the implicit `messages` that is passed. Could we also change the others in this file?